### DISCUSSION TITLE
QR Code

### DISCUSSION THREAD

**Discussion by imJohnYeager** [Auties00/WhatsappWeb4j#368] [Q&A]
When I run the code, it doesn't generate the QR code; instead, it simply terminates. Help me please...

![image](https://github.com/Auties00/Cobalt/assets/130397626/f3157294-b9fd-4177-b6e0-767255523cf5)

**Comment by Auties00** [comment]
> When I run the code, it doesn't generate the QR code; instead, it simply terminates. Help me please...
> 
> ![image](https://github.com/Auties00/Cobalt/assets/130397626/f3157294-b9fd-4177-b6e0-767255523cf5)
> 

That's normal, replace connect() with:
                .connectAwaitingLogout().join()

**Reply by imJohnYeager** [reply]
![image](https://github.com/Auties00/Cobalt/assets/130397626/05410bef-5793-4d91-ad46-154aceb8f6c4)
My script closes by itself...
